docs/api-conventions.md

```diff
@@ -84,27 +109,39 @@ ports:
 
 This rule maintains the invariant that all JSON/YAML keys are fields in API objects. The only exceptions are pure maps in the API (currently, labels, selectors, and annotations), as opposed to sets of subobjects.
 
+
 ### Lists
 
-Every list SHOULD have the following metadata in a nested object field called "metadata":
+Every list kind SHOULD have the following metadata in a nested object field called "metadata":
 
 * resourceVersion: a string that identifies the common version of the objects returned by in a list. This value MUST be treated as opaque by clients and passed unmodified back to the server. A resource version is only valid within a single namespace on a single kind of resource.
 
 
-Special Resources
------------------
+### Simple
+
+Every simple kind returned by the server, and any simple kind sent to the server that must support idempotency or optimistic concurency, SHOULD have the following metadata in a nested object field called "metadata":
+
+* resourceVersion: a string that identifies the common version of the objects returned by in a list. This value MUST be treated as opaque by clients and passed unmodified back to the server. A resource version is only valid within a single namespace on a single kind of resource.
```

Comment:
A simple object isn't necessarily a list. Maybe borrow the non-list-oriented explanation, or combine the two?